### DISCUSSION TITLE
Why Does Transcode Job Show More Videos Than I Have?

### DISCUSSION THREAD

**Discussion by coltenkrauter** [immich-app/immich#14753] [Q&A] [answered]
I have **15,724 videos** on my Immich server, but when I run the transcode job with **ALL**, it says,

```Waiting: 45,179```.

- Why is the job counting more videos than I have?  
- Is Immich creating multiple copies of each video when transcoding?  

I’d like to understand how this count is calculated and if duplicate transcoded files are being made.  

![Screenshot 2024-12-17 at 11 27 17 AM](https://github.com/user-attachments/assets/13861841-4e93-45fa-b7b5-6aef3c08b0dd)
![Screenshot 2024-12-17 at 11 28 19 AM](https://github.com/user-attachments/assets/692aca23-48de-4aef-ad63-1791077b2aaa)

**Comment by schuhbacca** [comment, marked as answer]
Do you have any external libraries? 
Do you have a lot of motion photos?

**Reply by coltenkrauter** [reply]
> Do you have any external libraries?

No

> Do you have a lot of motion photos?

Probably

**Reply by coltenkrauter** [reply]
Is there a way to find out how many motion photos I have?

**Reply by coltenkrauter** [reply]
![Screenshot 2024-12-17 at 11 41 12 AM](https://github.com/user-attachments/assets/e58abae4-e6a5-4639-bf2b-0d01e7c8cb9c)

I am seeing a ton of DEBUG logs about missing embedding. Not sure if that helps at all. (Notre I have other jobs running so this may be totally unrelated.)

**Reply by schuhbacca** [reply]
I am unsure if there is a way to see just motion photo counts without a DB query.

The log for embedding is related to Smart Search. Unrelated to transcode.

**Reply by coltenkrauter** [reply]
I have 165k photos in my library... I would be surprised if it turns out that I have ~30k motion photos. But it is possible.

**Reply by schuhbacca** [reply]
Live photos are on by default for Iphones and some Android devices. Might not be that crazy. But like I said hard to say.

**Reply by coltenkrauter** [reply]
You were correct. 

I made [this script](https://github.com/krauters/immich-utils/blob/main/src/total-motion-photos.ts), which hits the immich API to tally motion photos.

```
--- Report ---
Total motion photos found: [30048]
Elapsed time: [37.37] seconds
Search completed successfully
```

**Reply by mertalev** [reply]
For future reference, you can query this like so:
```sql
select count(*)
from assets
where type = 'VIDEO' and "livePhotoVideoId" is not null;
```

**Reply by coltenkrauter** [reply]
Good to know, ty
